baseplate/experiments/__init__.py

```diff
@@ -171,66 +183,19 @@ def variant(self, name, user=None, bucketing_event_override=None,
 
         if do_log:
             assert bucketing_id
-            self._log_bucketing_event(experiment, variant, user,
-                                      extra_event_fields)
+            if self._event_logger:
```

Comment:
You don't need this `if` anymore since you set it to the Debug logger if nothing is defined.

Reply:
fixed